geomesa-quickstart-kafka/pom.xml

```diff
@@ -11,9 +11,8 @@
     <modelVersion>4.0.0</modelVersion>
     <packaging>pom</packaging>
     <modules>
-        <!-- GeoMesa Kafka 0.9 and 0.10 not yet being distributed -->
-        <!--<module>geomesa-quickstart-kafka-10</module>-->
-        <!--<module>geomesa-quickstart-kafka-09</module>-->
+        <module>geomesa-quickstart-kafka-10</module>
+        <module>geomesa-quickstart-kafka-09</module>
```

Comment:
I would leave these commented out until we're distributing artifacts for kafka 0.9 and kafka 0.10. Right now somebody who runs ``mvn clean install`` in the root geomesa-tutorials directory will have their build fail because Maven can't find the geomesa-kafka-{09,10}-datastore dependencies.